jme3-plugins/src/gltf/java/com/jme3/scene/plugins/gltf/GltfLoader.java

```diff
@@ -1086,6 +1086,12 @@ public Joint readNodeAsBone(int nodeIndex, int jointIndex, int skinIndex, Matrix
 
     private void findChildren(int nodeIndex) throws IOException {
         JointWrapper jw = fetchFromCache("nodes", nodeIndex, JointWrapper.class);
+        
+        //Check if jw node exist or leave the function
+        if(jw == null){
```

Comment:
Can you please add a space between `){` for proper formatting?

Reply:
Also, the comment is explaining the code, which is simple enough to read without the comment. A good comment should not describe the code when the code can easily be read, it should describe why the code is there or what the code is doing from a abstract perspective. 

In this case I see little value added from a comment. Just my opinion.